tutorials/Clipboard.readHtml.md

```diff
@@ -0,0 +1,9 @@
+Read the content of the clipboard as Html
+# Example
+```js
+async function readHtml() {
+    return await fin.Clipboard.readHtml();
+}
+
+readHtml().then(stdout => console.log(stdout)).catch(err => console.log(err));
```

Comment:
stdout and stdin have special meaning, lets change the names here to not create confusion.